### ISSUE TITLE
Add support for launchy 3.0+

### ISSUE THREAD

**Issue by gaffneyc** [email-spec/email-spec#228]
We ran into an issue recently running `bundle update` where `email_spec` was downgraded to 0.6.6 due to launchy 3.0 being available. It was downgraded because we didn't have a version specifier for `email_spec` and 0.6.6 was the last version without a constraint on launchy.

It [doesn't look like](https://github.com/copiousfreetime/launchy/blob/main/HISTORY.md) there were too many breaking changes so it might just be a case of relaxing the constraint.

**Comment by sikachu** [comment]
I believe this should already be possible after https://github.com/email-spec/email-spec/pull/226 was merged. Would you mind try pointing to `email-spec` main branch on GitHub?

@etagwerker maybe we should cut 2.3.3, unless you have anything else you plan to include in the next release? Thanks!

**Comment by gaffneyc** [comment]
Yeah, you're right that it's fixed just waiting a release so going to close this out.
